tensorflow/python/kernel_tests/mkl_ops_test.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
```

Comment:
The docstring format should look like this:
```
"""One-line summary.

Longer description in a paragraph.
"""
```
For example, please see [Google Python Style Guide](http://google.github.io/styleguide/pyguide.html#381-docstrings).